src/Microsoft.Framework.ConfigurationModel.Xml/XmlConfigurationSource.cs

```diff
@@ -18,29 +18,57 @@ public class XmlConfigurationSource : ConfigurationSource
         private readonly XmlDocumentDecryptor _xmlDocumentDecryptor;
 
         public XmlConfigurationSource(string path)
-            : this(path, null)
+            : this(path, null, optional: false)
+        {
+        }
+
+        internal XmlConfigurationSource(string path, bool optional)
```

Comment:
Not sure about the history. But it looks to me this can be made public. I'll filed work item https://github.com/aspnet/Configuration/issues/168 to investigate it. This can be for beta5.